Makefile

```diff
@@ -7,8 +7,8 @@ install-tools:
 
 VERIFY += generate
 .PHONY: generate
-generate: CODE_GEN_DIR = pkg/json
-generate: GEN = pkg/cmd/codegenerator/generator.go
+generate: CODE_GEN_DIR = pkg/json/schemas
+generate: GEN = pkg/cmd/generator/generator.go
 generate:
 	go run $(GEN) -s ./json/ovn-nb.ovsschema -d $(CODE_GEN_DIR)
```

Comment:
on the second thought,  maybe we should rename ./json dir to ./schemas

Reply:
I meant at the top json directory - go run $(GEN) -s ./json/ovn-sb.ovsschema -d $(CODE_GEN_DIR) ->
go run $(GEN) -s ./schemas/ovn-sb.ovsschema -d $(CODE_GEN_DIR)

Reply:
no sure ...

Reply:
I'm talking about the input directory, where we store schemas(the `-s` flag of the  generator)

Reply:
ok moved form `json` to `schemas`